intro-to-R-tidyverse/03-intro_to_tidyverse-live.Rmd

```diff
@@ -439,7 +445,7 @@ This is what we will use the `file.path()` function for.
 Let's look in a bit more detail what `file.path()` does, by examining the 
 results of the function in the examples below.
 
-```{r}
+```r
```

Comment:
Curly brackets and name? (Ignore if you answered this elsewhere).

Reply:
This one I have more thoughts about... Keeping in mind that a goal is to have the non-live version run without errors... This one would not actually generate an error, but would generate some odd output. What do we want the output in the `nb.html` file to look like? If we leave it as a chunk, we would see all of the generated file paths, which might be fine. Or we could hide the output?